File: Ruby/first_vagrant_box/tdd/projectTDD/project.rb
class Project
    attr_accessor :name, :description, :owner, :tasks
    def initialize(name, description, owner)
        @name = name
        @description = description
        @owner = owner
        @tasks = []
    end
    def elevator_pitch
        "#{name}, #{description}, #{owner}"
    end
    def add_tasks(task)
        @tasks.push(task)
    end
    def tasks
        "#{@tasks}"
    end
end

File: java/javaOOP/projectClass/Project.java
import java.util.*;

public class Project {
    private String name;
    private String description;
    public Project() {
    }
    public Project(String name) {
    	this.name = name;
    }
    public Project(String name, String description) {
    	this.name = name;
    	this.description = description;
    }
    public String elevatorPitch() {
    	return "The project name is: " + name + " and the description is: " + description;
    }
    public String getName() {
    	return this.name;
    }
    public void setName(String name) {
    	this.name = name;
    }
    public String getDescription() {
    	return this.description;
    }
    public void setDescription(String description) {
    	this.description = description;
    }
}